features-json/document-exec-command.json

```diff
@@ -0,0 +1,224 @@
+{
+  "title":"Document.execCommand()",
+  "description":"Allows running commands to manipulate the contents of an editable region in a document switched to designMode",
+  "spec":"https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html#execcommand()",
+  "status":"wd",
```

Comment:
What's the code for Editor's Draft at https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md#supported-changes ?